Prepare v2.0.1 release (#113)
sighupio/fury-kubernetes-monitoring: docs/releases/v2.0.1.md

> # Monitoring Core Module Release 2.0.0

Reviewer: ```suggestion
# Monitoring Core Module Release 2.0.1
```

Welcome to the latest release of the `monitoring` core module of the [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution), maintained by team SIGHUP.

> This is a patch release including the following change:
> - added `GOMAXPROCS=1` envrionment variable to `node-exporter` to avoid throttling (see #106).

Reviewer: ```suggestion
This is a patch release including the following changes:
- added `GOMAXPROCS=1` envrionment variable to `node-exporter` to avoid throttling (see [#106](https://github.com/sighupio/fury-kubernetes-monitoring/issues/106)).
- aligned Service port name for `kube-controller-manager` and `kube-schedule` with the one expected by the ServiceMonitors (see [#109](https://github.com/sighupio/fury-kubernetes-monitoring/issues/109) )
```

| `kube-state-metrics`  | [`v2.5.0`](https://github.com/kubernetes/kube-state-metrics/releases/tag/v2.5.0)             | `No Update`      |
| `node-exporter`       | [`v1.3.1`](https://github.com/prometheus/node_exporter/releases/tag/v1.3.1)                  | `No Update`      |
| `prometheus-adapter`  | [`v0.9.1`](https://github.com/kubernetes-sigs/prometheus-adapter/releases/tag/v0.9.1)        | `No Update`      |
| `prometheus-operator` | [`v0.57.0`](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.57.0) | `No Update`      |
| `prometheus`          | [`v2.36.1`](https://github.com/prometheus/prometheus/releases/tag/v2.36.1)                   | `No Update`      |
| `thanos`              | [`v0.24.0`](https://github.com/thanos-io/thanos/releases/tag/v0.24.0)                        | `No Update`      |
| `x509-exporter`       | [`v3.2.0`](https://github.com/enix/x509-certificate-exporter/releases/tag/v3.2.0)            | `No Update`      |

> Please refer the individual release notes to get a detailed info on the releases.

## Update Guide 🦮

### Process

To upgrade this core module from `v2.0.0` to `v2.0.1`, execute the following:

```shell

> kustomize build vendor/node-exporter | kubectl apply -f --server-side

Reviewer: ```suggestion
kustomize build vendor/node-exporter | kubectl apply -f --server-side
kustomize build vendor/kubeadm-sm | kubectl apply -f --server-side
```
